src/components/UserUpdate/UserUpdate.test.tsx

```diff
@@ -79,12 +79,16 @@ describe('Testing User Update', () => {
   const props = {
     key: '123',
     id: '1',
+    visible: true,
   };
 
   const formData = {
-    firstName: 'Ansh',
-    lastName: 'Goyal',
-    email: 'ansh@gmail.com',
+    firstName: 'FName',
+    lastName: 'LName',
+    email: 'user@email.com',
+    password: 'password',
+    selectedOption: 'selectsuperadmin',
+    applangcode: 'en',
```

Comment:
The UserUpdate shouldn't have these new fields. There is a separate page for password update.